core/src/test/java/org/apache/calcite/rel/rel2sql/RelToSqlConverterTest.java

```diff
@@ -5430,6 +5430,43 @@ private void checkLiteral2(String expression, String expected) {
         .ok(expected);
   }
 
+
+  @Test void testTINYINTRedshift() {
```

Comment:
Reserved words are case insensitive so TINYINT vs. tinyint does not make a different for Calcite. Please remove this variant.

Reply:
Done !